mmcv/cnn/utils/weight_init.py

```diff
@@ -57,10 +65,399 @@ def caffe2_xavier_init(module, bias=0):
+
+    def __init__(self, gain=1, bias=0, distribution='normal', layers=None):
+        super().__init__(bias, layers)
+        self.gain = gain
+        self.distribution = distribution
+
+    def __call__(self, module):
+        if self.layers is None:
+            xavier_init(module, self.gain, self.bias, self.distribution)
+        else:
+            if isinstance(self.layers, str):
+                layername = module.__class__.__name__
+                if layername == self.layers:
+                    xavier_init(module, self.gain, self.bias,
+                                self.distribution)
+            elif isinstance(self.layers, list):
+                for layer in self.layers:
+                    layername = module.__class__.__name__
+                    if layername == layer:
+                        xavier_init(module, self.gain, self.bias,
+                                    self.distribution)
+            else:
+                raise TypeError(
+                    f'layers must be str or [str], but gor {type(self.layers)}'
+                )
+
+
+@INITIALIZERS.register_module(name='Normal')
+class NormalInit(BaseInit):
+    r"""Initialize module parameters with the values drawn from the normal
+    distribution :math:`\mathcal{N}(\text{mean}, \text{std}^2)`.
+
+    Args:
+        mean (int | float):the mean of the normal distribution.
+        Defaults to 0.
+        std (int | float): the standard deviation of the normal
+        distribution. Defaults to 1.
+        bias (int | float | dict): the value to fill the bias or
+        define initialization type for bias. Defaults to 0.
+        layers (str | [str], optional): the layer will be initialized.
+        Defaults to None.
+
+    """
+
+    def __init__(self, mean=0, std=1, bias=0, layers=None):
+        super().__init__(bias, layers)
+        self.mean = mean
+        self.std = std
+
+    def __call__(self, module):
+        if self.layers is None:
+            normal_init(module, self.mean, self.std, self.bias)
+        else:
+            if isinstance(self.layers, str):
+                layername = module.__class__.__name__
+                if layername == self.layers:
+                    normal_init(module, self.mean, self.std, self.bias)
+            elif isinstance(self.layers, list):
+                for layer in self.layers:
+                    layername = module.__class__.__name__
+                    if layername == layer:
+                        normal_init(module, self.mean, self.std, self.bias)
+            else:
+                raise TypeError(
+                    f'layers must be str or [str], but gor {type(self.layers)}'
+                )
+
+
+@INITIALIZERS.register_module(name='Uniform')
+class UniformInit(BaseInit):
+    r"""Initialize module parameters with values drawn from the uniform
+    distribution :math:`\mathcal{U}(a, b)`.
+
+    Args:
+        a (int | float): the lower bound of the uniform
+        distribution. Defaults to 0.
```

Comment:
May check the doc in the local host to see whether the results are expected. Usually we will leave an indent when switching lines for an argument